Resources/Prototypes/_NF/Research/industrial.yml

```diff
@@ -0,0 +1,33 @@
+﻿# Tier 1
+
+# Tier 2
+
+- type: technology
+  id: RapidConstruction
+  name: research-technology-rapid-construction
+  icon:
+    sprite: Objects/Tools/rcd.rsi
+    state: icon
+  discipline: Industrial
+  tier: 2
+  cost: 10000
+  recipeUnlocks:
+  - RCD
+  - RCDAmmo
+  - BorgModuleRCD
+
+# Tier 3
+
+# - type: technology # frontier - threw this in here since I noticed it was missing, incase anyone wanted it back in. not the point of this pr so can remove it if needed.
+#   id: CompactPower
+#   name: research-technology-compact-power
+#   icon:
+#     sprite: Structures/Power/Generation/wallmount_generator.rsi
+#     state: panel
+#   discipline: Industrial
+#   tier: 3
+#   cost: 15000
+#   recipeUnlocks:
+#   - WallmountSubstationElectronics
+#   - WallmountGeneratorElectronics
+#   - WallmountGeneratorAPUElectronics
```

Comment:
If this was uncommented on upstream file and missing, put it in upstream file, if not, remove this uncommented from here as it has no use to be here.

Reply:
It was removed entirely from upstream, but still exists as a recipe to be unlocked by the Circuit printer via research

Reply:
This should be fully removed here tbh it makes power more free then its already is.

Reply:
Yeah I do definitely agree that power is cheap right now.

I'm just sad to see usable content being cut is all. Maybe it can see a return someday as like a tier 3 Civilian tech or something. I'll get it nixed after work